middelwares/authentication.js

```diff
@@ -0,0 +1,81 @@
+require("dotenv").config();
+const jwtService = require('../service/jwt-service')
+const userService = require('../service/user-service')
+
+const verifyAuth = async (request, response, next) => {
+    console.log("In verify auth");
+    const authHeader = request.headers.access_token;
+    console.log(authHeader)
+    if (authHeader) {
+        const token = authHeader.split(".")[1];
+
+        console.log("token :", token);
+
+        jwtService.verifyToken(
+            authHeader,
```

Comment:
authHeader should be replaced with token

Reply:
Not working with token